examples/language-modeling/run_mlm_flax.py

```diff
@@ -520,14 +560,15 @@ def generate_batch_splits(samples_idx: jnp.ndarray, batch_size: int) -> jnp.ndar
     else:
         column_names = datasets["validation"].column_names
     text_column_name = "text" if "text" in column_names else column_names[0]
+    columns_to_remove = [column_name for column_name in column_names if column_name != text_column_name]
```

Comment:
This new variable is not used below AFAICT since `column_names` is used in the `map` method. (This makes me wonder how the CI is passing since flake8 usually isn't happy with that).